# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: 31587f0a98cd
# Full Hash: 31587f0a98cd6b7a445c59ccd18c2207259dafd2
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-03-14 09:41:39
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Share all-of-dcomp.h preamble, and deal with outdated mingw dcomp.h. r=gfx-reviewers,sotaro
#   
#   Mingw's dcomp.h is not the official one, but rather a by-hand
#   reproduction. While this newly-updated version has e.g.
#   IDCompositionFilterEffect, it is still missing e.g.
# ==============================================================================

diff -r ab2b044c704c -r 31587f0a98cd gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Mon Mar 13 21:04:12 2023 +0000
+++ b/gfx/thebes/moz.build	Mon Mar 13 21:04:12 2023 +0000
@@ -67,6 +67,7 @@
 ]
 
 EXPORTS.mozilla.gfx += [
+    "AllOfDcomp.h",
     "D3D11Checks.h",
     "DeviceManagerDx.h",
     "DisplayConfigWindows.h",